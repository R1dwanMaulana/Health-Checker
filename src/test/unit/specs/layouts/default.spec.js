import { mount, createLocalVue, RouterLinkStub } from "@vue/test-utils";
import Vuetify from "vuetify";

import Default from "~/layouts/default";

const localVue = createLocalVue();
localVue.use(Vuetify);

describe("Default layout", () => {
  let vuetify;
  beforeEach(() => {
    vuetify = new Vuetify();
  });
  it("renders to match snapshot", () => {
    const wrapper = mount(Default, {
      localVue,
      vuetify,
      stubs: {
        nuxt: "nuxt",
        "router-link": RouterLinkStub,
        "nuxt-link": RouterLinkStub,
        "nuxt-child": "nuxt-child"
      }
    });
    expect(wrapper).toMatchSnapshot();
  });
});
