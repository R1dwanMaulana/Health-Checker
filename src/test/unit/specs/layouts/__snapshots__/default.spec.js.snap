// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default layout renders to match snapshot 1`] = `
<div data-app="true" class="v-application v-application--is-ltr theme--light" id="inspire">
  <div class="v-application--wrap">
    <nav class="v-navigation-drawer v-navigation-drawer--close v-navigation-drawer--fixed v-navigation-drawer--is-mobile theme--light" style="height: 100vh; top: 0px; transform: translateX(-100%); width: 256px;">
      <div class="v-navigation-drawer__content">
        <div class="v-list v-sheet v-sheet--tile theme--light v-list--dense"><a tabindex="0" class="v-list-item v-list-item--link theme--light">
            <div class="v-list-item__action"><i aria-hidden="true" class="v-icon notranslate mdi mdi-home theme--light"></i></div>
            <div class="v-list-item__content">
              <div class="v-list-item__title">Home</div>
            </div>
          </a> <a tabindex="0" class="v-list-item v-list-item--link theme--light">
            <div class="v-list-item__action"><i aria-hidden="true" class="v-icon notranslate mdi mdi-airballoon theme--light"></i></div>
            <div class="v-list-item__content">
              <div class="v-list-item__title">Inspire</div>
            </div>
          </a></div>
      </div>
      <div class="v-navigation-drawer__border"></div>
    </nav>
    <header class="v-sheet v-sheet--tile theme--dark v-toolbar v-app-bar v-app-bar--fixed indigo" style="height: 64px; margin-top: 0px; transform: translateY(0px); left: 0px; right: 0px;">
      <div class="v-toolbar__content" style="height: 64px;"><button type="button" class="v-app-bar__nav-icon v-btn v-btn--flat v-btn--icon v-btn--round theme--dark v-size--default"><span class="v-btn__content"><i aria-hidden="true" class="v-icon notranslate material-icons theme--dark">$menu</i></span></button>
        <div class="v-toolbar__title">Application</div>
      </div>
    </header>
    <main class="v-content" style="padding: 64px 0px 0px 0px;">
      <div class="v-content__wrap">
        <div class="container container--fluid fill-height">
          <div class="layout justify-center align-center">
            <div class="flex">
              <div></div>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer class="v-footer v-sheet v-sheet--tile theme--light v-footer--fixed indigo" style="left: 0px; right: 0px; bottom: 0px;"><span class="white--text">Â© 2020</span></footer>
  </div>
</div>
`;
